### ISSUE TITLE
Put function's start accolade on new line

### ISSUE THREAD

**Issue by JuJuDropThor** [gossi/php-code-generator#49]
Hi guys,

thanks for that lib, pretty usefull.

Is it possible to generate function like that ?

```
public function setEmail($email)
{
	$this->email = $email;
	return $this;
}
```

instead of that 

```
public function setEmail($email) {
	$this->email = $email;
	return $this;
}
```

I 'd like the first accolade be on new line.

Is that possible ? I didn't find any parameter. Maybe overriding something ?

I got the same issue on the accolade of my class.

class User {

instead of

Class User
{

**Comment by JuJuDropThor** [comment]
Maybe an issue here

protected function writeFunctionStatement(RoutineInterface $model) {
		$this->writer->write('function ');
		
		if ($model->isReferenceReturned()) {
			$this->writer->write('& ');
		}
		
		$this->writer->write($model->getName() . '(');
		$this->writeParameters($model);
		$this->writer->write(')');
		$this->writeFunctionReturnType($model);
	}

**Comment by gossi** [comment]
The strategy towards this is roughly outlined in #23

**Comment by gossi** [comment]
Implemented, see #54
